### ISSUE TITLE
LighthouseIntegration does not report all the traces to Sentry

### ISSUE THREAD

**Issue by DenisKeyprod** [getsentry/sentry-laravel#882]
### How do you use Sentry?

Sentry SaaS (sentry.io)

### SDK version

4.2.0

### Steps to reproduce

1. Setup a simple Laravel/Lighthouse server with a random field (for ie "query.todos") and use "Laravel Sentry" library (https://docs.sentry.io/platforms/php/guides/laravel/) with a correct DSN and SENTRY_TRACES_SAMPLE_RATE=1.0
2. Perform a query to you server, using a fragment defined at the beginning of your query. For instance
```graphql
fragment todoFields on ToDo {
  id
  description
}

query todos{
  todos {
    ...todoFields
  }
}
```

### Expected result

The query you made to your server should appear in your Sentry project under "Performances>Transactions>lighthouse?query{todos}"

### Actual result

No such event appear in Sentry

**Comment by stayallive** [comment]
Do you get any event or just completely nothing? Is it possibly you are you over your quota for your organization possibly resulting in transaction not being reported, or do you get other transactions from the same application just fine?

Also to double check, did you clear or re-cache your config `php artisan config:clear` and/or `php artisan config:cache` if you use config caching?

Does `php artisan sentry:test --transaction -vvv` work and show no errors?

**Comment by DenisKeyprod** [comment]
`php artisan sentry:test --transaction -vvv` does not return any error.

The culprit is "src/Sentry/Laravel/Tracing/Integrations/LighthouseIntegration.php" and its method "handleStartExecution".
At line 130, there is a test that check if the first operation definition is a OperationDefinitionNode but when the "fragment" is not one.

**Comment by stayallive** [comment]
That is interesting, but does make sense... thanks for the info!

**Comment by cleptric** [comment]
We released [4.5.0](https://github.com/getsentry/sentry-laravel/releases) that includes a fix for this issue.
